### PR TITLE
Display Base Address + Reward funds when selecting the owner wallet f…

### PR DESCRIPTION
**Pull request by TrevorBenson**
…or pledge and showing wallets.

Some default within my IDE appears to have adjusted a bunch of blank lines. I can track down the setting and remove it and resubmit.

### CODE REVIEW

File: scripts/cnode-helper-scripts/cntools.sh
Comment (rdlrt) [inline scripts/cnode-helper-scripts/cntools.sh:759]: Best to hold this part out on specific screen until HD wallet support as it will be inaccurate regardless of what we enter as field here

File: scripts/cnode-helper-scripts/cntools.sh
Comment (rdlrt) [inline scripts/cnode-helper-scripts/cntools.sh:2285]: $(( lovelace + reward_lovelace ))

Comment (TrevorBenson) [reply scripts/cnode-helper-scripts/cntools.sh]: Thanks. I've familiar with multiple methods to add integers, but which is new/old and benefits of each I don't keep committed to memory, which I should :+1:

**Review by rdlrt — APPROVED** [review]

**Review by Scitz0 — APPROVED** [review]
LGTM but would also like to extend it to include a change in cntools.library wallet selection function as well as pool pledge verification checks. But I will do this after merging PR.

File: scripts/cnode-helper-scripts/cntools.sh
Comment (TrevorBenson) [reply scripts/cnode-helper-scripts/cntools.sh]: Meant to respond back to this and inquire for knowledge sake. What is the inaccuracy? Something in the way we calculate the balances I would assume since as this is just addition.

Comment (rdlrt) [reply scripts/cnode-helper-scripts/cntools.sh]: What we should be aiming for would be "Total Delegated" and that would encompass HD Wallet balance + Rewards. Currently a Funds + Rewards gives possibly a wrong picture as your wallet could have a lot more balance on addr_ix > 1 than what is seen by CNTools

**Pull request merged**
